--- configure.ac.orig	2021-05-06 21:25:52.520752918 +0800
+++ configure.ac	2021-05-06 21:26:02.612768380 +0800
@@ -19,7 +19,7 @@
 # so that we create the export library with the dll.
 #-----------------------------------------------------------------------
 
-AC_INIT([tclsignal], [1.4])
+AC_INIT([tclsignal], [1.4.4])
 
 #--------------------------------------------------------------------
 # Call TEA_INIT as the first TEA_ macro to set up initial vars.
